### ISSUE TITLE
Could you please provide the training schedule for ADE20k dataset?

### ISSUE THREAD

**Issue by zbwxp** [dvlab-research/Entity#8]

**Comment by qqlu** [comment]
For a fair comparison, there are two settings to train ADE20K including the similar training epoch or similar training iteration to the COCO dataset (12 epochs or 90000 iterations with the batch size 16). For ADE20K with thoroughly 20000 images, the iteration could be set 15000 iterations (12*20000/16) or 90000 iterations. The learning rate decays 0.1 at 10000 and 13333 iterations if choosing 15000 iterations. The other hyperparameter settings are similar to COCO.

We will release this part of the codes as soon as possible.
